### DISCUSSION TITLE
Script to do TTS (text to Speech) of node text in Windows

### DISCUSSION THREAD

**Discussion by biswajit-goswami** [freeplane/freeplane#1703] [Ideas]
Anyone has explored or tried a script that can invoke the Microsoft TTS commands to read out load the text in a node? If not, any hint how that could be implemented?

**Comment by euu2021** [comment]
This Autohotkey script: when the user presses F1, it does a `Ctrl+C`, and whatever is copied is read aloud:

```
#SingleInstance, force
f1::
send ^c
ComObjCreate("SAPI.SpVoice").Speak(Clipboard)
return
```

You may want to use, instead of `Ctrl+C`, your hotkey for `Edit->Copy->Copy node (single)`, so it doesn't read the whole branch.
